test/sanitizer_suppressions/tsan

```diff
@@ -39,5 +30,6 @@ race:libzmq
 # https://github.com/bitcoin/bitcoin/issues/20618
 race:CZMQAbstractPublishNotifier::SendZmqMessage
 
-# https://github.com/bitcoin/bitcoin/pull/20218, https://github.com/bitcoin/bitcoin/pull/20745
-race:epoll_ctl
+# Uninstrumented external libraries.
+called_from_lib:libdb
+called_from_lib:libevent
```

Comment:
Maybe still remove if it is not needed?

Reply:
Ok. I'll push an updated branch after the current one finishes its CI jobs (to be sure in it).

Reply:
It didn't pass locally, fyi

Reply:
> It didn't pass locally, fyi

The current branch or without `called_from_lib:libevent`?

Reply:
The branch as is, IIRC